src/System.Data.Common/tests/System/Data/Common/DataColumnMappingCollectionTest.cs

```diff
@@ -472,7 +472,7 @@ public void AddRange_Array_PassingNullThrowsException()
             Array array = null;
             DataColumnMappingCollection dataColumnMappingCollection = new DataColumnMappingCollection();
 
-            Assert.Throws<ArgumentNullException>(() => dataColumnMappingCollection.AddRange(array));
+            Assert.Throws<ArgumentNullException>("values", () => dataColumnMappingCollection.AddRange(array));
```

Comment:
I believe we're now trying to avoid Assert.Throws when a parameter name is supplied and instead use AssertExtensions.Throws.  The latter avoids the parameter name check on platforms where it might be optimized away in release builds.